docs/customizing.md

```diff
@@ -159,9 +163,111 @@ stages:
 Note the installation hooks only cover installation procedures, for upgrades equivalent
 `before-upgrade` and/or `after-upgrade-chroot` should be defined.
 
+### Adding extra LVM volume group disks during the installation
+
+This example is covering the case the host has multiple disks, some of them used
+as part of an LVM setup.
+
+In this particular case we will assume a host includes three disks (`/dev/sda`, `/dev/sdb`
+and `/dev/sdc`), where the first one is used for a regular Elemental Teal installation
+and the other remaining two are used as part of LVM group where abitrary logical volumes
+are created, formatted and mounted at boot time via an extended fstab file.
+
+For that use case the following files are required:
+
+* additional clout-init files to include within the installed system
+* additional installation hooks to prepare the LVM volumes at install time
+* additional Elemental client configuration file to point hooks file location
+
+Lets create an `overlay` directory to create the directory tree that needs to be
+added into the ISO root. In that case the `overlay` directory could contain:
+
+```yaml showLineNumbers
+overlay/
+  oem/
+    lvm_volumes_in_fstab.yaml
+  hooks/
+    lvm_volumes_hook.yaml
+  elemental/
+    config.yaml
+```
+
+The Elemental client config file in `overlay/elemental` could be as:
+
+```yaml showLineNumbers
+cloud-init-paths:
+  - "/run/initramfs/live/hooks"
+```
+
+This is just to let Elemental client know where to find installation hooks.
+
+The installation hook `overlay/hooks/lvm_volumes_hook.yaml` could be as:
+
+```yaml showLineNumbers
+name: "Create LVM logic volumes over some physical disks"
+stages:
+  post-install:
+    - name: "Create physical volume, volume group and logical volumes"
+      if: '[ -e "/dev/vdb" ] && [ -e "/dev/vdc" ]'
```

Comment:
above we assume /dev/sda, /dev/sdb and /dev/sdc . Here we are using /dev/vdb and /dev/vdc

Reply:
oh you are right 😓 Yes I did the tests under KVM and used virtio devices.